utils/language_utils.py

```
@@ -0,0 +1,785 @@
+        Returns:
+            翻译结果或 None（超时或失败时返回 None）
+        """
+        if not GOOGLETRANS_AVAILABLE:
+            return None
+        
+        try:
+            translator = Translator()
+            
+            # 使用 asyncio.wait_for 实现超时机制
+            async def _translate_internal():
+                # 如果文本太长（超过15k字符），分段翻译
```

Ruff (0.14.10)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)